components/notebook-controller/Dockerfile

```diff
@@ -6,11 +6,10 @@
 #
 # This is necessary because the Jupyter controller now depends on
 # components/common
-ARG GOLANG_VERSION=1.12.5
+ARG GOLANG_VERSION=1.13
```

Comment:
Why change the `go` version as part of this PR. IMO that should be a separate PR